<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchForm</class>
 <widget class="QWidget" name="SearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="searchWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchPushButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="PropertiesGroupBox" name="propertiesGroupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DataTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QWidget" name="closeWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataTableWidget</class>
   <extends>QTableWidget</extends>
   <header>DataTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>PropertiesGroupBox</class>
   <extends>QGroupBox</extends>
   <header>PropertiesGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
